package peer;

message RegisterServer {
		required string server_name = 1;
		required int32 server_id = 2;
		required string param = 3;
}